resources/services/ec2/images.go

```diff
@@ -305,3 +311,28 @@ func fetchEc2ImageBlockDeviceMappings(ctx context.Context, meta schema.ClientMet
 	res <- r.BlockDeviceMappings
 	return nil
 }
+
+func resolveEc2ImageLastLaunchedTime(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	image := resource.Item.(types.Image)
+	svc := cl.Services().EC2
+	opts := ec2.DescribeImageAttributeInput{
+		Attribute: types.ImageAttributeNameLastLaunchedTime,
+		ImageId:   image.ImageId,
+	}
+	result, err := svc.DescribeImageAttribute(ctx, &opts, func(options *ec2.Options) {
+		options.Region = cl.Region
+		options.EndpointResolver = ec2.EndpointResolverFromURL(fmt.Sprintf("https://ec2.%s.amazonaws.com", cl.Region))
+	})
+	if err != nil {
+		return nil
```

Comment:
Is there a reason why we are not wrapping this error?

Reply:
Good catch, I still haven't used to do it. BTW return value is wrong here :D